### PR TITLE
Hide image and tagView in donations and Petitions

### PR DESCRIPTION
**Pull request by mahmoudashraf93**
Task: https://trello.com/c/vzrIV43B

After removing the image

Simulator Screen Shot - iPhone 11 - 2020-06-11 at 00 44 20

### CODE REVIEW

File: SayTheirNames/Source/View/CallToAction/CallToActionCell.swift [hunk 1]
```diff
@@ -33,7 +33,7 @@ final class CallToActionCell: UICollectionViewCell {
         $0.translatesAutoresizingMaskIntoConstraints = false
         $0.contentMode = .scaleAspectFill
         $0.clipsToBounds = true
-        
+        $0.isHidden = true
```
Comment (ohwhen) [inline SayTheirNames/Source/View/CallToAction/CallToActionCell.swift:36]: nice work 👍  any chance we can put this behind a FeatureFlag? There's already a precedent used in other places in the code base.

Comment (mahmoudashraf93) [reply SayTheirNames/Source/View/CallToAction/CallToActionCell.swift:36]: Sure I'll add the feature flag now

Comment (mahmoudashraf93) [reply SayTheirNames/Source/View/CallToAction/CallToActionCell.swift:36]: Would something like this be fine `$0.isHidden = !FeatureFlags.callToActionCellImageShown`

Comment (mahmoudashraf93) [reply SayTheirNames/Source/View/CallToAction/CallToActionCell.swift]: Done

**Pull request merged**